### PR TITLE
tests: reduce flakiness

### PR DESCRIPTION
**Pull request by Nirajn2311**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: e2e/editor-drawer.spec.ts [hunk 1]
```diff
@@ -100,9 +101,9 @@ test.describe("feature image", () => {
     // Check that the post was saved successfully
     const saveNotificationTitle = page.locator("#toast-1-title");
     await expect(saveNotificationTitle).toBeVisible();
-    expect(await saveNotificationTitle.innerText()).toBe(
-      "Post has been updated.",
-    );
+    await expect(saveNotificationTitle).toHaveText("Post has been updated.", {
+      useInnerText: true,
+    });
```
Comment (ojeytonwilliams) [inline e2e/editor-drawer.spec.ts:106]:
```suggestion
    await expect(saveNotificationTitle).toHaveText("Post has been updated.");
```
This seemed to work just fine for me. Was it necessary to have `useInnerText` on your machine?

File: e2e/editor-drawer.spec.ts [hunk 2]
```diff
@@ -140,9 +141,9 @@ test.describe("feature image", () => {
     // Wait for the save notification to appear
     const saveNotificationTitle = page.locator("#toast-1-title");
     await expect(saveNotificationTitle).toBeVisible();
-    expect(await saveNotificationTitle.innerText()).toBe(
-      "Post has been updated.",
-    );
+    await expect(saveNotificationTitle).toHaveText("Post has been updated.", {
+      useInnerText: true,
+    });
```
Comment (ojeytonwilliams) [inline e2e/editor-drawer.spec.ts:146]:
```suggestion
    await expect(saveNotificationTitle).toHaveText("Post has been updated.");
```

File: e2e/editor-drawer.spec.ts [hunk 3]
```diff
@@ -42,13 +42,14 @@ test.describe("feature image", () => {
     const saveNotificationTitle = page.locator("#toast-1-title");
     const saveNotificationDescription = page.locator("#toast-1-description");
     await expect(saveNotificationTitle).toBeVisible();
-    expect(await saveNotificationTitle.innerText()).toBe(
-      "Post has been updated.",
-    );
+    await expect(saveNotificationTitle).toHaveText("Post has been updated.", {
+      useInnerText: true,
+    });
```
Comment (ojeytonwilliams) [inline e2e/editor-drawer.spec.ts:47]:
```suggestion
    await expect(saveNotificationTitle).toHaveText("Post has been updated.");
```

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]

File: e2e/editor-drawer.spec.ts (hunk 1 above)
Comment (Nirajn2311) [reply e2e/editor-drawer.spec.ts:106]: Not exactly. Playwright dint work properly when I checked yesterday so I pushed them here to check them through actions. Lemme clean up and try the tests again.

**Review by ojeytonwilliams — APPROVED** [review]
Cheers, @Nirajn2311, this LGTM :+1:

**Pull request merged**